src/main/scala/firrtl/WIR.scala

```diff
@@ -147,6 +151,7 @@ class WrappedType(val t: Type) {
       case (_: UIntType, _: UIntType) => true
       case (_: SIntType, _: SIntType) => true
       case (ClockType, ClockType) => true
+      case (_: AnalogType, _: AnalogType) => false
```

Comment:
That's correct. This is how Analog totally skips out of the Firrtl type system. The only way Analog can play with another Analog component is through Attach. Otherwise, we'd need to special case it during ExpandWhens, Lowering, ExpandConnects, etc.